dev-python/pipenv/pipenv-2020.6.2-r1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+MY_PV=${PV/_beta/b}
+DESCRIPTION="Python Development Workflow for Humans"
+HOMEPAGE="https://github.com/pypa/pipenv https://pypi.org/project/pipenv/"
+SRC_URI="https://github.com/pypa/pipenv/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}"/${PN}-${MY_PV}
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/certifi[${PYTHON_USEDEP}]
+	>=dev-python/pip-18.0[${PYTHON_USEDEP}]
+	dev-python/virtualenv[${PYTHON_USEDEP}]
+	>=dev-python/virtualenv-clone-0.2.5[${PYTHON_USEDEP}]
+	dev-python/six
```

Comment:
Why are you not using `[${PYTHON_USEDEP}]` here?

Reply:
Please see my comments in [the issue ](https://bugs.gentoo.org/738132)regarding these dependencies.

Reply:
Added PYTHON_USEDEP on the new dependencies as needed.

Reply:
Thanks. I'm still not sure why these dependencies are added. See my comments.
Maybe someone else can back me or contradict me.

Reply:
@dave77 can you point us to upstream dependency requirements, or show an error log if pipenv fails without these deps?
Ideally you'd first created a bug about it.

Reply:
I wasn't sure if it was related to a different bug because there was a report that dev-lang/pipenv-2020.6.2 fails tests (https://bugs.gentoo.org/738132), so I didn't open a new bug when I couldn't run pipenv.

I've now created this bug describing the problem: https://bugs.gentoo.org/748138

When I install pipenv via pip (appdirs and six both get pulled in).  The stack traces show these packages not being found.